Barotrauma/BarotraumaShared/SharedSource/Items/Components/ItemContainer.cs

```diff
@@ -458,6 +462,22 @@ public override void Update(float deltaTime, Camera cam)
                 alwaysContainedItemsSpawned = true;
             }
 
+            float totalConditionValue = 0;
+            float totalConditionPercentage = 0;
+            int totalItems = 0;
+            foreach (var item in Inventory.AllItems)
+            {
+                if (item.Condition != 0)
+                {
+                    totalConditionValue += item.Condition;
+                    totalConditionPercentage += item.ConditionPercentage;
+                    totalItems++;
+                }
+            }
+            item.SendSignal(totalConditionValue.ToString(), "contained_conditions");
+            item.SendSignal(totalConditionPercentage.ToString(), "contained_condition_percentages");
+            item.SendSignal(totalItems.ToString(), "contained_items");
```

Comment:
The ToString calls need to use invariant culture, otherwise the values will be different depending on the culture.

Reply:
Fixed.